### PR TITLE
Fix turbine

### LINKED ISSUE #89: Creating unphysical instances of WindTurbine class
With the current API nothing prevents a user from creating a nonphysical wind turbine instance, i.e., wind turbine that has *hub_height* shorter than 0.5*_rotor_diameter_. I would suggest raising ```ValueError``` with a useful message in case this happens, what do you think?

I wrote a small fix that passes all the existing test and adds new. If this is something you would like to review, and see fixed, I could easily put it in a PR?

### PR DESCRIPTION
**Pull request by vezeli**
Fixes #89 .

### CODE REVIEW

File: windpowerlib/wind_turbine.py [hunk 1]
```diff
@@ -163,6 +163,11 @@ def __init__(self, hub_height, nominal_power=None, path='oedb',
                 if self.rotor_diameter is None and turbine_data is not None:
                     self.rotor_diameter = float(turbine_data['rotor_diameter'])
 
+        if self.rotor_diameter:
+            if self.hub_height <= 0.5*self.rotor_diameter:
+                msg = "1/2*rotor_diameter cannot be higher than hub_height"
+                raise ValueError(msg)
```
Comment (uvchik) [inline windpowerlib/wind_turbine.py]: I am not a native speaker but as I know it in a mathematical sense one value should not be GREATER than another not HIGHER.

File: tests/test_wind_turbine.py [hunk 2]
```diff
@@ -97,3 +97,13 @@ def test_wrongly_defined_to_group_method(self):
...
+
+    def test_create_unphysical_turbine(self):
+        char = {
+           'hub_height': 80,
+           'rotor_diameter': 160,
+           'turbine_type': 'DUMMY 3',
+           'path': self.source
+        }
+        with pytest.raises(ValueError):
+            WindTurbine(**char)
```
Comment (uvchik) [inline tests/test_wind_turbine.py]: I would use the match attribute to make sure the right ValueError is raised.

```python
 with pytest.raises(ValueError, match=regex)
```

File: windpowerlib/wind_turbine.py (hunk 1 above)
Comment (vezeli) [reply windpowerlib/wind_turbine.py]: Changed.

File: tests/test_wind_turbine.py (hunk 2 above)
Comment (vezeli) [reply tests/test_wind_turbine.py]: Implemented. However, I had a problem with match when there was an asterisk symbol in the string so I modified the error message slightly.

**Pull request merged**